pkg/db/vulnerability.go

```diff
@@ -24,13 +24,16 @@ func (dbc Config) GetVulnerability(cveID string) (vuln types.Vulnerability, err
 	err = db.View(func(tx *bolt.Tx) error {
 		bucket := tx.Bucket([]byte(vulnerabilityBucket))
 		value := bucket.Get([]byte(cveID))
+		if value == nil {
+			return xerrors.New("no item in the database for this vulnerability")
+		}
```

Comment:
`vulnerability` bucket contains extra details for vulns: severity, description, score etc...
i think that `no item in the database` isn't an accurate description of this problem.

also we (trivy's team) should clarify a reason for such situations.

so I'm not sure that this new error makes sense.

Reply:
@afdesk Thank you for the explanation. I changed it to `no extra details for this vulnerability`, how does that sound?

The full error logged by trivy will end up being: `Error while getting vulnerability details: failed to get the vulnerability "CVE-1234-123456": no extra details for this vulnerability`